src/vs/workbench/contrib/search/browser/media/searchview.css

```diff
@@ -33,6 +33,7 @@
 .search-view .search-widget .monaco-inputbox > .wrapper > textarea.input {
 	padding: 3px;
 	padding-left: 4px;
+	padding-right: 22px;
```

Comment:
This will be applied to every input box in the search widget, right? This should only apply to the replace box. Please just find where we do this for the search box already and do exactly the same thing for replace.

Reply:
Everything looks good to me , I don't see any thing out of place.
[This](https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/search/browser/media/searchview.css#L63) is where css for replace box input as I said before and adding `padding-right` is moving it out of input box and not working.
I guess for search input box the css is applied differently because of the way it is rendered and replace box is rendered differently.
SearchInput rendered [here](https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/search/browser/searchWidget.ts#L290)
Replace Input rendered [here](https://github.com/microsoft/vscode/blob/master/src/vs/workbench/contrib/search/browser/searchWidget.ts#L343)
One basic diff I would find is one uses `attachFindInputBoxStyler` and one more used `attachInputBoxStyler`

Please let me know what I can do next here ?

PS : This is how it looks now

<img width="1440" alt="Screen Shot 2019-09-17 at 10 40 41 PM" src="https://user-images.githubusercontent.com/17094832/65063616-3e02b080-d99c-11e9-92b8-f49b85a3a6b8.png">